src/Chainweb/Chainweb/ChainResources.hs

```diff
@@ -155,7 +157,7 @@ withChainResources v cid rdb peer logger mempoolCfg cdbv payloadDb prune dbDir n
                 logg Info $ "finished pruning block header database. Deleted " <> sshow x <> " block headers."
 
             -- replay pact
-            let pact = pes mempool requestQ
+            let pact = pes requestQ
```

Comment:
This is used only once in line 168. Inline?

Reply:
Done